tests/unit/constraints/test_tabular.py

```diff
@@ -3998,7 +3993,7 @@ def test_reverse_transform_numerical_data(self):
             'retirement_age': [65, 68, 75],
             'current_age': [21, 22, 25],
         })
-        expected_transform = transform(
+        expected_transform = logit(
```

Comment:
For the unit tests, let's just make sure `logit` and `sigmoid` are called with the correct parameters and we use whatever they return correctly. Then we can add unit tests for those functions specifically